src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import "./Introduce.scss";
+import { useNavigate } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+
+const Introduce = () => {
+
+    const navigate = useNavigate();
+
+    const goToProduct = () => {
+        navigate('/product');
+    };
+
+    return (
+    <div className = "box">
+        <div className = "wecodebox">
+            <h1>>wecode</h1>
+        </div>
+        <p>유정인</p>
+        <p>43기</p>
+        <div>
+            <input type="text"></input>
```

Comment:
```suggestion
            <input type="text" />
```

JSX에서는 self closing이 가능합니다.
추가로 input에 event가 없네요! intro 과제 보시면서 추가해주세요!